docs/guide/repository.md

```diff
@@ -484,209 +467,74 @@ ezpublish:
     system:
         # "site_group" refers to the group configured in site access
         site_group:
-            # "default" refers to the cache pool, the one configured on stash.caches above
-            cache_pool_name: "default"
+            # by default cache service is set to cache.app via parameter '%env(CACHE_POOL)%'
+            cache_service_name: '%env(CACHE_POOL)%'
```

Comment:
Note, this has been updated in the code, it is now: `cache_service_name: '%cache_pool%'`

And in turn `cache_pool: '%env(CACHE_POOL)%'`

Change was done here: https://github.com/ezsystems/ezplatform/pull/233